docs/DEVELOPER_GUIDE.md

```diff
@@ -333,5 +333,12 @@ Ideally, the `version` specified should match the release tag of the plugin.
 This helps users and maintainers to easily identify which version of the plugin
 they have installed.
 
+### Automating the releasing of new version
+
+You can use `github actions` to automate the release process. e.g. You can use
+`go-releaser` (or equivalent for other languages) for creation of new release and 
+[`krew-release-bot`](https://github.com/rajatjindal/krew-release-bot) to bump the 
+version in `krew-index` repo. Refer to the [documentation here](https://github.com/rajatjindal/krew-release-bot#basic-setup) for an example.
```

Comment:
No need for #target link. 
Just link to your repo home. It should be concise enough for someone to understand how it works in 60 seconds or so. :)

Reply:
done